docs/env-profile.mdx

```diff
@@ -0,0 +1,87 @@
+---
+id: env-profile
+title: Environment profile
+sidebar_position: 8
+---
+
+# Environment profile
+
+To mitigate AI unpredictability, we have implemented several strategies to enhance its behavior and reliability. This page outlines the environment-specific details attached to chat sessions.
```

Comment:
Can we rephrase it a bit? IMO unpredictability isn't relevant here, it's more about response quality? 🤔 
 
```suggestion
To enhance AI response quality and reliability, we have implemented several strategies. This page outlines one of them—the environment-specific details attached to Fuse chat sessions.
```